packages/neuron-wallet/src/block-sync-renderer/index.ts

```diff
@@ -38,11 +38,13 @@ const syncNetwork = async (rescan = false) => {
   LockUtils.cleanInfo()
   DaoUtils.cleanInfo()
 
-  const blockNumber = new BlockNumber()
+  const blockNumber = new SyncedBlockNumber()
   if (rescan) {
     await blockNumber.reset()
   }
-  SyncedBlockNumberSubject.getSubject().next((await blockNumber.getCurrent()).toString())
+  const startBlockNumber = (await blockNumber.getNextBlock()).toString()
```

Comment:
Does `getNextNumber` or `getNextBlockNumber` a better name ?

Reply:
Not a big problem to me, as class name `SyncedBlockNumber` already tells this is `block number` related.

Reply:
ok